src/server/status/index.js

```diff
@@ -53,6 +56,9 @@ export default function (kbnServer, server, config) {
   server.route(wrapAuth({
     method: 'GET',
     path: '/status',
+    config: {
+      tags: ['api']
```

Comment:
I agree, I had to get that tag in to /status to receive a 401 when accessing the api programatically, although I was still forced to go through a redirect first:


```
GET /api/status HTTP/1.1
Connection: Keep-Alive
Content-Type: application/json
Content-Length: 0
Host: localhost:5601
User-Agent: Manticore 0.6.1
Accept-Encoding: gzip,deflate

HTTP/1.1 302 Found
location: /kxg/status
vary: origin,accept-encoding
cache-control: no-cache
content-length: 0
Date: Tue, 18 Jul 2017 15:35:45 GMT
Connection: keep-alive


GET /kxg/status HTTP/1.1
Connection: Keep-Alive
Content-Type: application/json
Host: localhost:5601
User-Agent: Manticore 0.6.1
Accept-Encoding: gzip,deflate


HTTP/1.1 401 Unauthorized
kbn-name: kibana
kbn-version: 6.0.0-beta1
vary: origin,accept-encoding
content-type: application/json; charset=utf-8
cache-control: no-cache
content-encoding: gzip
date: Tue, 18 Jul 2017 15:35:46 GMT
connection: keep-alive
transfer-encoding: chunked
```